[2021-10-15 00:00:02,598] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_B 2021-10-14T00:00:00+00:00 [queued]>
[2021-10-15 00:00:02,613] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_B 2021-10-14T00:00:00+00:00 [queued]>
[2021-10-15 00:00:02,613] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-15 00:00:02,614] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-15 00:00:02,615] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-15 00:00:02,630] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): Training_Model_B> on 2021-10-14T00:00:00+00:00
[2021-10-15 00:00:02,638] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Training_Model_B', '2021-10-14T00:00:00+00:00', '--job-id', '196', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpzxyz4r1r', '--error-file', '/tmp/tmpwgmssyud']
[2021-10-15 00:00:02,634] {standard_task_runner.py:52} INFO - Started process 38257 to run task
[2021-10-15 00:00:02,640] {standard_task_runner.py:77} INFO - Job 196: Subtask Training_Model_B
[2021-10-15 00:00:02,692] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Training_Model_B 2021-10-14T00:00:00+00:00 [running]> on host d6b4131456ad
[2021-10-15 00:00:02,762] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Training_Model_B
AIRFLOW_CTX_EXECUTION_DATE=2021-10-14T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-14T00:00:00+00:00
[2021-10-15 00:00:02,784] {python.py:151} INFO - Done. Returned value was: None
[2021-10-15 00:00:02,795] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Training_Model_B, execution_date=20211014T000000, start_date=20211015T000002, end_date=20211015T000002
[2021-10-15 00:00:02,872] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-15 00:00:02,904] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
